.gitlab-ci.yml

```diff
@@ -1,10 +1,25 @@
+---
+# -*- coding: utf-8 -*-
+
+image: golang
```

Comment:
It's probably better to specify the same go version as in .travis.yml  through the image tag (`golang:1.10`).
Otherwise the checksum of the GitHub release and GitLab release binary will diverge.

Reply:
Ok, fair enough.